cluster-autoscaler/core/placeholderpod/types.go

```diff
@@ -0,0 +1,47 @@
+package placeholderpod
+
+import (
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+)
+
+// Tolerations is a slice of k8s tolerations
+type Tolerations []apiv1.Toleration
+
+// Spec is a specification of a placeholder pod
+type Spec struct {
```

Comment:
There is already a pod spec in the api. Please do not introduce another structure for the same purpose.